spring-boot-daemon-sample/src/main/java/net/nicoll/SampleApplication.java

```diff
@@ -16,25 +13,37 @@
 
 package net.nicoll;
 
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+@EnableScheduling
```

Comment:
I am not keen to use scheduling for the sample. That sounds like quite an agressive option to me.